README.rst

```diff
@@ -69,6 +69,254 @@ Windows
     <your-env>\Scripts\pip.exe install google-cloud-aiplatform
 
+
+Overview
+~~~~~~~~
+This section provides a brief overview of the Vertex SDK for Python. You can also reference the notebooks in `vertex-samples`_ for examples.
+
+.. _vertex-samples: https://github.com/GoogleCloudPlatform/ai-platform-samples/tree/master/ai-platform-unified/notebooks/unofficial/sdk
+
+Importing
+^^^^^^^^^
+SDK functionality can be used from the root of the package:
+
+.. code-block:: Python
+
+    from google.cloud import aiplatform
+
+
+Initialization
+^^^^^^^^^^^^^^
+Initialize the SDK to store common configurations that will be used throughout the SDK.
+
+.. code-block:: Python
+
+    aiplatform.init(
+        # your GCP project ID or number
+        # environment default used is not set
+        project='my-project',
+
+        # the Vertex AI region you will use
+        # defaults to us-central1
+        location='us-central1',
+
+        # bucket in same region as location
+        # used to stage artifacts
+        staging_bucket='gs://my_staging_bucket',
+
+        # custom google.auth.credentials.Credentials
+        # environment default creds used if not set
+        credentials=my_credentials,
+
+        # customer managed encryption key resource name
+        # will be applied to all AI Platform resources if set
+        encryption_spec_key_name=my_encryption_key_name,
+
+        # the name of the experiment to use to track
+        # logged metrics and parameters
+        experiment='my-experiment',
+
+        # description of the experiment above
+        experiment_description='my experiment decsription' 
+    )
+
+Datasets
+^^^^^^^^
+AI Platform provides managed Tabular, Text, Image, and Video datasets. In the SDK, Datasets can be used downstream to
+train models.
+
+To create a Tabular dataset:
+
+.. code-block:: Python
+
+    my_dataset = aiplatform.TabularDataset.create(
+        display_name="my-dataset", gcs_source=['gs://path/to/my/dataset.csv'])
+
+You can also create and import a dataset in separate steps:
+
+.. code-block:: Python
+
+    from google.cloud import aiplatform
+
+    my_dataset = aiplatform.TextDataset.create(
+        display_name="my-dataset")
+
+    my_dataset.import(
+        gcs_source=['gs://path/to/my/dataset.csv']
+        import_schema_uri=aiplatform.schema.dataset.ioformat.text.multi_label_classification
+    )
+
+To get a previously created Dataset:
+
+.. code-block:: Python
+  
+  dataset = aiplatform.ImageDataset('projects/my-project/location/us-central1/datasets/{DATASET_ID}')
+
+Vertex AI supports a variety of dataset schemas. References to these schemas are available under the
+:code:`aiplatform.schema.dataset` namespace. For more information on the supported dataset schemas please refer to the
+`Preparing data docs`_.
+
+.. _Preparing data docs: https://cloud.google.com/ai-platform-unified/docs/datasets/prepare
+
+Training
+^^^^^^^^
+The Vertex SDK allows you train Custom and AutoML Models.
```

Comment:
Insert 'AI' after "Vertex."

Reply:
Changed to Vertex SDK for Python. See here: https://cloud.google.com/vertex-ai/docs/start/client-libraries#python